### ISSUE TITLE
Toggle buttons in parameters vanish when enabled

### ISSUE THREAD

**Issue by molivier-fcms** [stefan-niedermann/nextcloud-deck#967]
**Describe the bug**

In parameters menu, if we toggle a switch to enable, the switch vanishes.

1. Go to parameters
2. Click on any toggle to enable it
3. It disappears / becomes blank

**Expected behavior**
It should display the toggle activated on right (direction)

**Screenshots**
When a switch is **OFF**

![off](https://user-images.githubusercontent.com/39309715/116975653-45d1ae80-acc0-11eb-8a86-8c1460d5c99e.jpg)

When a switch is **ON**

![on](https://user-images.githubusercontent.com/39309715/116975666-48340880-acc0-11eb-8c3f-41897042ead4.jpg)

**Versions**
 - Nextcloud:  20.0.6
 - Nextcloud Deck: 1.18.2 (the UI language is french, in case it has an importance)
 - Nextcloud Android: 3.15.1
 - Nextcloud Android Deck: 1.2.6

**Smartphone (please complete the following information):**
- Device: Galaxy A40
- Android-Version: 11
- App-Store:
  - [ ] Google Play Store
  - [ ] Google Play Store (Beta channel)
  - [X] F-Droid
  - [ ] Huawei AppGallery

**Stacktrace**
None

**Comment by stefan-niedermann** [comment]
What theme color has your Nextcloud instance applied?

**Comment by molivier-fcms** [comment]
Right. Only light theme is affected. Dark theme displays the toggle on.

**Comment by molivier-fcms** [comment]
Nextcloud instances lay have another theme color? I guess I have default one so (light).

On Deck android application however I see only light theme is affected. Dark theme displays the toggles on.

**Comment by stefan-niedermann** [comment]
>  default theme (light)

It's nothing about light / dark theme, but on the **theme color**.

I suspect a theming issue, and i guess that a white is your applied theme color.

To verify, please share a screenshot of
- the Android app's sidebar header (with the logo)
  ![grafik](https://user-images.githubusercontent.com/4741199/116985482-d2826980-accc-11eb-976c-1fd292cd261b.png)

- The upper area of your Deck app in the web browser (where the Nextcloud logo is)
  ![grafik](https://user-images.githubusercontent.com/4741199/116985052-453f1500-accc-11eb-8cb9-861685791d66.png)

If you administrating your Nextcloud yourself, please open Nextcloud in the Web UI, click on your avatar in the top right corner -> Settings -> Theming (in the left sidebar) and share a screenshot like this:

![grafik](https://user-images.githubusercontent.com/4741199/116985251-81727580-accc-11eb-89de-41ab1568449e.png)

If you are only a user and do not administrate your Nextcloud instance yourself, please ask your administrator to do so.

**Comment by molivier-fcms** [comment]
Stefan,

Your intuition was right. In my case android app banner was hidden, I did not suspect its existence. I thought it was a kind of vertical centering for the menu.
![Screenshot_20210504-164340_Nextcloud Deck](https://user-images.githubusercontent.com/39309715/117023675-9ca7aa00-acf9-11eb-839f-0c489fc920dc.jpg)

The thing is: few months ago I disabled apps that did not look necessary. And theming app was part of it. However I have not encountered any color issue anywhere else without it, and top toolbar in Nextcloud displayed well.

Having brought the app back, everything is solved : 
![Screenshot_20210504-164750_Nextcloud Deck](https://user-images.githubusercontent.com/39309715/117024013-e98b8080-acf9-11eb-8e50-791235ff9c2d.jpg)
![Screenshot_20210504-164753_Nextcloud Deck](https://user-images.githubusercontent.com/39309715/117024032-ebedda80-acf9-11eb-8791-e2550076d51f.jpg)

Thank you for your diagnostic.
